spec/lib/tasks/properties_spec.rb

```diff
@@ -1,80 +1,71 @@
-require 'rake'
-require 'stringio'
+require "rake"
+require "stringio"
 
-describe 'properties rake tasks', live: true do
+describe "properties rake tasks", live: true do
   before :all do
-    Rake.application.rake_require 'tasks/properties'
+    Rake.application.rake_require "tasks/properties"
   end
 
-  context 'Contact Properties' do
-    let(:file) { '/tmp/contact-demo-props.json' }
-    let(:hapikey) { 'demo' }
+  let(:contact_file) { "/tmp/contact-demo-props.json" }
+  let(:demo_file) { "/tmp/deal-demo-props.json" }
+  let(:hapikey) { "demo" }
 
-    describe 'hubspot:dump_properties' do
-      let :run_rake_task do
-        Rake::Task['hubspot:dump_properties'].reenable
-        Rake.application.invoke_task "hubspot:dump_properties[contact,#{file},#{hapikey}]"
-      end
+  context "Contact Properties" do
+    describe "hubspot:dump_properties" do
+      it "writes the class properties and groups to the given file" do
+        VCR.use_cassette("fetch_contact_properties_and_groups") do
```

Comment:
@cbisnett These tests and the logic around writing JSON responses to files (`dump_properties`) and comparing a file with a JSON response (`hubspot:restore_properties`) would benefit from further refactoring but I'm uncertain how useful these tasks are to users and perhaps we can remove them instead of refactoring them. 

To play it safe and to push this PR along, I updated the tests just enough to avoid real API calls from happening.

Reply:
This work was extracted to a separate PR for easier review. See PR #142